tests/Dommel.Bulk.IntegrationTests/Dommel.Bulk.IntegrationTests.csproj

```diff
@@ -8,6 +8,7 @@
     </PropertyGroup>
 
     <ItemGroup>
+        <PackageReference Include="Microsoft.Data.Sqlite" Version="6.0.5" />
```

Comment:
There are two popular sqlite connectors. System.Data.SQLite.Core and Microsoft.Data.Sqlite.Core. I think it is necessary to write unit tests using both libraries